bitnami/sealed-secrets/templates/role.yaml

```
@@ -57,3 +64,61 @@ rules:
   {{- include "common.tplvalues.render" ( dict "value" .Values.rbac.serviceProxier.rules "context" $ ) | nindent 2 }}
   {{- end }}
 {{- end }}
+{{ if and (and .Values.rbac.create .Values.rbac.namespacedRoles) (not $.Values.rbac.clusterRole) }}
+---
+  {{- range $additionalNamespace := $.Values.additionalNamespaces }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ printf "%s" (include "sealed-secrets.namespacedRoleName" .) }}
+  namespace: {{ $additionalNamespace }}
+  labels: {{- include "common.labels.standard" ( dict "customLabels" .Values.commonLabels "context" $ ) | nindent 4 }}
+  {{- if .Values.rbac.labels }}
+  {{- include "common.tplvalues.render" ( dict "value" .Values.rbac.labels "context" $) | nindent 4 }}
+  {{- end }}
+  {{- if .Values.commonAnnotations }}
+  annotations: {{- include "common.tplvalues.render" ( dict "value" .Values.commonAnnotations "context" $ ) | nindent 4 }}
```

These are inside the range's context so we'll need to modify them as so:

```suggestion
  labels: {{- include "common.labels.standard" ( dict "customLabels" $.Values.commonLabels "context" $ ) | nindent 4 }}
  {{- if $.Values.rbac.labels }}
  {{- include "common.tplvalues.render" ( dict "value" $.Values.rbac.labels "context" $) | nindent 4 }}
  {{- end }}
  {{- if $.Values.commonAnnotations }}
  annotations: {{- include "common.tplvalues.render" ( dict "value" $.Values.commonAnnotations "context" $ ) | nindent 4 }}
```

Same suggestion for the `clusterrole.yaml` template.